components/MovieCard/IMovieData.tsx

```diff
@@ -17,6 +17,10 @@ interface IMovieGenre {
   name: string;
 }
 
+/* Movie Data Interface ==================
```

Comment:
- For interface declaration you do not need JSX. So change the extension from tsx to ts
- These are types not data. Call the file `MovieCard.types.ts`